exec/csnex/Bytecode.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace csnex
+{
+    public class Bytecode
+    {
+        private byte[] obj;
+        public string source_path;
+        private byte[] source_hash;
+        private uint version;
+        public uint global_size;
+        public List<String> strtable;
+        private uint strtablesize;
+        public byte[] code;
+        public uint codelen;
```

Comment:
This doesn't seem to be required. The length of the code is just the length of the `code` array, you don't have to keep track of the length separately.

Reply:
Removed.

Reply:
Removed.